edge-util/src/Microsoft.Azure.Devices.Edge.Storage/disk/IDiskSpaceChecker.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.Disk
+{
+    public interface IDiskSpaceChecker
```

Comment:
>IDiskSpaceChecker [](start = 21, length = 17)

Since I'm considering using the same interface and base classes for the MemorySpaceChecker, it might be better to rename these files to something like IStorageSpaceChecker or something similar? Essentially change Disk to Storage (or something else)?